kubernetes/kfserving/kf_request_json/bert_v2.json

```diff
@@ -0,0 +1,14 @@
+{
+  "inputs": [{
+    "name": "input-0",
+    "shape": [1, 37],
+    "datatype": "INT64",
+    "data": [ 66, 108, 111, 111, 109,  98, 101, 114, 103,  32, 104,  97, 115,  32, 114, 101, 112, 111, 114, 116, 101, 100,  32, 111, 110,  32, 116, 104, 101,  32, 101,  99, 111, 110, 111, 109, 121 ]
```

Comment:
What do api user's need to do to convert text inputs into this format? It will be good to add a utility function to make it easy for people to convert and invoke